hw/ip/ascon/data/ascon.hjson

```diff
@@ -0,0 +1,696 @@
+        Loaded into the internal Full Key register upon starting encryption/decryption.
+        All key registers (Share 0 and Share 1) must be written at least once when the key is changed.
+        The order in which the registers are updated does not matter.
+        Can only be updated when the Ascon unit is idle.
+        If the Ascon unit is non-idle, writes to these registers are ignored.
+        Upon reset, these registers are cleared with pseudo-random data.
+      '''
+      count: "NumRegsKey",
+      cname: "KEY_SHARE1",
+      swaccess: "wo",
+      hwaccess: "hrw",
+      hwext:    "true",
+      hwqe:     "true",
+      fields: [
+        { bits: "31:0", name: "key_share1", desc: "Initial Key Share 1" }
+      ],
+      }
+    },
+##############################################################################
+# input nonce registers
+    { multireg: {
+      name: "NONCE_SHARE0",
+      desc: '''
+        Input Nonce Register Share 0.
+        The actual data corresponds to Nonce Input Registers Share 0 XORed with Nonce Input Registers Share 1.
+        If SW does not want to provide the Nonce masked, it can simply set one share to all zeros and provide the unmasked nonce in the other share.
+        All nonce registers (Share0 and Share 1) must be written each time a new message is processed.
+        Ascon requires the nonce to be unique for each message. However there are no hardware checks to enforce this.
+        The order in which the registers are updated does not matter.
+        If the user fails to update the register the NO_NONCE flag is set in the ERROR Register. Can only be updated when the Ascon unit is idle.
+        If the Ascon unit is non-idle, writes to these registers are ignored. Upon reset, these registers are cleared with pseudo-random data.
+      '''
+      count: "NumRegsNonce",
+      cname: "NONCE_SHARE0",
+      swaccess: "wo",
+      hwaccess: "hrw",
+      hwext:    "true"
+      hwqe:     "true",
+      fields: [
+        { bits: "31:0", name: "nonce", desc: "Input Nonce Share 0" }
+      ],
+      }
+    },
+    { multireg: {
+      name: "NONCE_SHARE1",
+      desc: '''
+        Input Nonce Register Share 1.
+        The actual data corresponds to Nonce Input Registers Share 0 XORed with Nonce Input Registers Share 1.
+        If SW does not want to provide the Nonce masked, it can simply set one share to all zeros and provide the unmasked nonce in the other share.
+        All nonce registers (Share0 and Share 1) must be written each time a new message is processed.
+        Ascon requires the nonce to be unique for each message. However there are no hardware checks to enforce this.
+        The order in which the registers are updated does not matter.
+        If the user fails to update the register the NO_NONCE flag is set in the ERROR Register. Can only be updated when the Ascon unit is idle.
```

Comment:
You can reference this field by writing !!ERROR.NO_NONCE.

To cross check the rendered doc, you can run `./util/site/build-docs.sh serve`

Reply:
at the moment the documentation cannot be build. I think this is due to some missing hooks. I will take care of that after https://github.com/lowRISC/opentitan/pull/22262 is merged